### PR TITLE
Fullstack env

### PR DESCRIPTION
**Pull request by techlab-lainio**
- full stack env first version

### CODE REVIEW

File: scripts/fullstack/README.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+   ```
+   The Bob's terminal should output Alice's welcoming messages. To stop Bob's
+   listen command just press C-c.
+
+   **TODO: What else we really need for now?**
+
+   Should we have issuing example, chat bots, all of them or should we wait for
+   the feedback first and bring them if there is demand.
+
+1. [Optional, requires Node.js] Start wallet development environment. On the
```
Comment (lauravuo-techlab) [inline scripts/fullstack/README.md:169]: Remove this or add pointers to PWA repository?

File: README.md [hunk 2]
```diff
@@ -40,6 +40,14 @@ $ ./install.sh -b /usr/local/bin
 ```
 Don't forget to set the [auto-completion](#Shell-auto-completion)
 
+## Running Full Stack Agency
+
+For running quickly a full-stack trial please see the
+[guide](https://raw.githubusercontent.com/findy-network/findy-agent-cli/HEAD/scripts/fullstack/README.md).
```
Comment (lauravuo-techlab) [inline README.md:46]: Use relative path instead? `./scripts/fullstack/README.md`

**Review by lauravuo-techlab — APPROVED** [review]

File: scripts/fullstack/README.md (hunk 1 above)
Comment (techlab-lainio) [reply scripts/fullstack/README.md]: Thanks 👍

File: README.md (hunk 2 above)
Comment (techlab-lainio) [reply README.md]: thanks 👍

**Pull request merged**